### PR TITLE
fix(DDS): delete dds instance param validation

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  delete validation
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  delete validation
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/dds/' TESTARGS='-run TestAccDDSV3Instance_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dds/ -v -run TestAccDDSV3Instance_ -timeout 360m -parallel 4
=== RUN   TestAccDDSV3Instance_basic 
=== PAUSE TestAccDDSV3Instance_basic                       
=== RUN   TestAccDDSV3Instance_withEpsId                   
=== PAUSE TestAccDDSV3Instance_withEpsId                   
=== RUN   TestAccDDSV3Instance_prePaid                     
=== PAUSE TestAccDDSV3Instance_prePaid                     
=== RUN   TestAccDDSV3Instance_withConfigurationSharding   
=== PAUSE TestAccDDSV3Instance_withConfigurationSharding   
=== RUN   TestAccDDSV3Instance_withConfigurationReplicaSet 
=== PAUSE TestAccDDSV3Instance_withConfigurationReplicaSet 
=== RUN   TestAccDDSV3Instance_withConfigurationSingle     
=== PAUSE TestAccDDSV3Instance_withConfigurationSingle     
=== CONT  TestAccDDSV3Instance_basic
=== CONT  TestAccDDSV3Instance_withConfigurationSingle
=== CONT  TestAccDDSV3Instance_withConfigurationSharding
=== CONT  TestAccDDSV3Instance_withConfigurationReplicaSet
--- PASS: TestAccDDSV3Instance_withConfigurationSingle (687.96s) 
=== CONT  TestAccDDSV3Instance_prePaid
--- PASS: TestAccDDSV3Instance_withConfigurationReplicaSet (720.42s) 
=== CONT  TestAccDDSV3Instance_withEpsId
--- PASS: TestAccDDSV3Instance_withConfigurationSharding (723.64s) 
--- PASS: TestAccDDSV3Instance_withEpsId (806.08s) 
--- PASS: TestAccDDSV3Instance_basic (2494.55s) 
--- PASS: TestAccDDSV3Instance_prePaid (1864.04s) 
PASS 
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dds       2552.046s
```

### CODE REVIEW

**Comment by niuzhenguo** [comment]
Needs to update docs as well.

File: docs/resources/dds_instance.md [hunk 1]
```diff
@@ -177,13 +177,14 @@ The following arguments are supported:
...
 
 * `version` - (Required, String, ForceNew) Specifies the DB instance version. For the Community Edition, the valid
-  values are 3.2, 3.4, or 4.0. For the Enhanced Edition, only 3.4 is supported now.
+  values are `3.2`, `3.4`, `4.0`, `4.2`, or `4.4`. For the Enhanced Edition, only `3.4` is supported now.
 
-* `storage_engine` - (Optional, String, ForceNew) Specifies the storage engine of the DB instance. DDS Community Edition
-  supports wiredTiger engine, and the Enhanced Edition supports rocksDB engine.
+* `storage_engine` - (Optional, String, ForceNew) Specifies the storage engine of the DB instance.
+  If `version` is set to `3.2`, `3.4`, or `4.0`, the value is **wiredTiger**.
+  If `period_unit` is set to `4.2`, or `4.4`, the value is *rocksDB**.
```
Comment (ShiChangkuo) [inline docs/resources/dds_instance.md]:
```suggestion
  If `period_unit` is set to `4.2`, or `4.4`, the value is **rocksDB**.
```
format error

Comment (houpeng80) [reply docs/resources/dds_instance.md]: fixed

**Pull request merged**